tools/bin/ci_performance_test.sh

```diff
@@ -7,8 +7,6 @@ set -e
 # runs performance tests for an performance name
 
 connector="$1"
-firstarg=""
-secondarg=""
 if [[ "$2" ]]; then
   if [[ "$2" == *"cpulimit"* ]]; then
     firstarg="-DcpuLimit=$(echo $2 | cut -d / -f 2)"
```

Comment:
The current way to specify the parameters is a bit redundant. Can we do it like this:

```bash
/test-performance connector=connectors/source-mysql cpu=4 memory=2GB
```

Since most of the parameters are cut out, I think the following statement should work:

```suggestion
    firstarg="-DcpuLimit=$(echo $2 | cut -d = -f 2)"
```

Reply:
I tried to do it as you indicated, but there are several points that do not allow us to simplify the work with the parameters.

Reply:
I tried to do it as you indicated, but there are several points that do not allow us to simplify the work with the parameters.
1. In the example that you indicated in the comment, **cpu** and **memory** are the names of the inputs in the github. Only their values will be transferred to the shell script - **4** and **2GB**, respectively. Without the names of the github inputs. Therefore, we cannot parse the input parameters of the shell script through the = sign.
2. Both parameters are optional and I have provided the ability to name only one of the parameters. For example, we can change the size of the container's memory limit, but not change the CPU limit. This means that not the CPU limit, but the memory limit will be passed to the shell script as the second parameter, and the third parameter will be empty.

Reply:
@tuliren, sorry, I accidentally posted an unfinished comment without clarification. =)